README.md

```diff
@@ -37,6 +37,7 @@ This is useful for notifying multiple groups, or casually asking for help.
 
 These are 3rd-party plugins that are included in this bot:
 
+- [changelog](https://github.com/mikz/probot-changelog): Automatically ensure that the `CHANGELOG` is updated for every Pull Request (configured in the repos `/.github/changelog.yml`)
```

Comment:
would be nice if this wasn't a separate config, but a stanza in the other .github/config.yml But I guess this is a touch-once-per-repo sort of thing

Reply:
yeah I agree. I can try to submit a PR so that the bot optionally looks for `.github/config.yml` **and** uses https://github.com/getsentry/probot-config so we have less copy/pasta in the repos (`probot-config` offers a way to "inherit" a config from another repo)